src/snowflake/snowpark/_internal/data_source/drivers/base_driver.py

```diff
@@ -166,6 +163,18 @@ def data_source_data_to_pandas_df(
 
     @staticmethod
     def to_result_snowpark_df(
-        session: "Session", table_name, schema, _emit_ast: bool = True
+        session: "Session", table_name: str, schema: StructType, _emit_ast: bool = True
```

Comment:
we have `_emit_ast = True` but we do not have `@publicapi`

Reply:
see my comment above